pkg/proto/configuration/scheduler/scheduler.proto

```diff
@@ -88,6 +92,18 @@ message DemultiplexingActionRouterConfiguration {
   ActionRouterConfiguration default_action_router = 3;
 }
 
+message RemoteActionRouterConfiguration {
+  // gRPC client configuration.
```

Comment:
In the remote authenticator, there is a scope field so that the server can know who is connecting. Shall we add that pattern here as well?